sorc/orog_mask_tools.fd/inland.fd/inland.F90

```diff
@@ -68,6 +68,8 @@ PROGRAM inland_mask
 
 CONTAINS
 
```

Comment:
We need two docblocks at the top of the file, one for the file, one for the program. Like this:

```
!> @file   
!! @brief Some brief description of file.
!! @author Freddy Fortran

!> A brief description of the program.
!! @author Freddy Fortran
!! @return 0 for success, error otherwise.
program prog1
```

Reply:
Still need two docblocks at the top of the file.

Use a completely blank line to separate them. See my example above.